src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -230,6 +230,7 @@ private function publishOnExchange(\AMQPExchange $exchange, string $body, string
     {
         $attributes = $amqpStamp ? $amqpStamp->getAttributes() : [];
         $attributes['headers'] = array_merge($attributes['headers'] ?? [], $headers);
+        $attributes['delivery_mode'] = $attributes['delivery_mode'] ?? 2;
```

Comment:
for readability purpose, shouldn't we use `\AMQP_DURABLE` instead of an arbitrary int?

Reply:
I'm not sure `AMQP_DURABLE` means `delivery_mode=2`. 
`AMQP_DURABLE` is for the queue & exchange durability. Not the data
See the [documentation](https://github.com/pdezwart/php-amqp/blob/master/stubs/AMQP.php#L16-L19)